### PR TITLE
8686 Panzer: Adding WorksetOptions to Workset during setup

### PR DESCRIPTION
**Pull request by seamill**
@trilinos/<teamName>

### CODE REVIEW

**Review by rppawlo — DISMISSED** [review]

File: packages/panzer/adapters-stk/test/panzer_workset_builder/apply_orientations.cpp [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+#include "Panzer_STK_Interface.hpp"
+#include "Panzer_STK_SquareQuadMeshFactory.hpp"
+#include "Panzer_STKConnManager.hpp"
+#include "Panzer_STK_WorksetFactory.hpp"
+
+namespace panzer {
+namespace {
+
+void
+testApplyOrienations(const bool by_container,
```
Comment (rppawlo) [inline packages/panzer/adapters-stk/test/panzer_workset_builder/apply_orientations.cpp:68]: orientations is misspelled in the function name.

**Review by rppawlo — APPROVED** [review]
this is a really good test @seamill ! Thanks!

File: packages/panzer/adapters-stk/test/panzer_workset_builder/apply_orientations.cpp (hunk 1 above)
Comment (seamill) [reply packages/panzer/adapters-stk/test/panzer_workset_builder/apply_orientations.cpp:68]: I made that spelling error a dozen times at least while making these changes... I'll fix it.

**Comment by seamill** [comment]
@rppawlo I'll have to hold off on merging this. Somehow these changes broke empire.

**Review by rppawlo — APPROVED** [review]

**Pull request merged**